README.md

```diff
@@ -26,9 +26,8 @@ This repository contains a `CMakeLists.txt` with an example project that depends
 
 ```bash
 cd cmake-conan
-mkdir build
-cmake -B build -S . -DCMAKE_PROJECT_TOP_LEVEL_INCLUDES=conan_provider.cmake -DCMAKE_BUILD_TYPE=Release
-cmake --build build --config Release
+cmake --preset default
```

Comment:
Let's leave both approaches.
I think it is good that we provide both, with and without presets, for users that don't want to use presets. Could you please leave it?